### ISSUE TITLE
m2m (oidc_client) applications support

### ISSUE THREAD

**Issue by IsmaelMartinez** [okta/okta-sdk-python#286]
Hello,

I would like to programmatically generate oidc_client applications in okta. 

When I try to list them, I get the following response back:
```
'DYNAMIC' is not a valid OpenIdConnectApplicationIssuerMode
```

The same happens if I try to do a get_application

Here is an example code:

```
async def main():
    async with OktaClient(config) as client:
        # perform all queries within same session
        # list apps
        apps, resp, err = await client.list_applications()
        print(apps)
        print(resp)
        print(err)
        app, resp, err = await client.get_application('myclientIdGoesHere')
        print(app)
        print(resp)
        print(err)
```

I have created the application following the next steps:
* go to the application section and click "Create App Integration"
* select "API Services" and click "next"
* click "save" with the default name (I also tried with a name without spaces)

My aim is to support creating the app configuration as code for machine to machine services, removing the manual steps that introduce incertainty and are error prone.

Thanks in advance for your help.

**Comment by IsmaelMartinez** [comment]
I have validated this and it works in the [nodejs sdk](https://github.com/okta/okta-sdk-nodejs). I would prefer to use python as it is the prefered language in our company but I will continue with my investigation wit nodejs

**Comment by MasterSergius** [comment]
@IsmaelMartinez Looks like, "DYNAMIC" is a not supported issuer mode so far - https://github.com/okta/okta-sdk-python/blob/master/okta/models/open_id_connect_application_issuer_mode.py
Just for testing purposes, you can try add this manually (into installed okta package) and see if it works for you. There might be few options how to resolve this issue.

**Comment by bretterer** [comment]
Resolved in 2.9.0
